documentation/remarkable2.md

```diff
@@ -0,0 +1,53 @@
+# Booting OE Linux on the reMarkable2
+
+The [reMarkable2](https://remarkable.com/) is an e-ink reader using the iMX7
+SoC.
+
+This layer adds support for building the official kernel source and u-boot
+source for the rM2.
+
+In order to build the images for the rM2 setup this layer and all dependencies
+and then run:
+
+```shell
+MACHINE = "imx7d-remarkable2" bitbake core-image-minimal -k
```

Comment:
I am not against to add the doc, but it ought to match the machine name.

Reply:
Fixed.